<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>597</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="seeProject">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background: transparent</string>
         </property>
         <property name="text">
          <string>See Projects</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="seeHomeWork">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <italic>false</italic>
           <bold>true</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background: transparent</string>
         </property>
         <property name="text">
          <string>See Home Work</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addHomeWork">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Add Home Work</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addProject">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background: transparent</string>
         </property>
         <property name="text">
          <string>Add Projects</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="contentTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add Home Work</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>35</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter Home Work</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="anyButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Add Home Work</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>200</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="deleteLast_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteLast">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Delete Last</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
color: white;
background-color: #0F0F0F;
border-radius: 25px;
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1001</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">olor: white;
font-size: 50px;</string>
    </property>
    <property name="text">
     <string>The App For Students!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title3">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>140</y>
      <width>1001</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
color: #E8E8E8;</string>
    </property>
    <property name="text">
     <string>Presented By Berke...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color :#E8E8E8;</string>
    </property>
    <property name="text">
     <string>Click To Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="title2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">olor: white;</string>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
